packages/tool-cache/src/tool-cache.ts

```diff
@@ -202,9 +202,36 @@ export async function extractTar(
     throw new Error("parameter 'file' is required")
   }
 
+  // Create dest
   dest = await _createExtractFolder(dest)
-  const tarPath: string = await io.which('tar', true)
-  await exec(`"${tarPath}"`, [flags, '-C', dest, '-f', file])
+
+  // Determine whther GNU tar
+  let versionOutput = ''
+  await exec('tar --version', [], {
+    ignoreReturnCode: true,
+    listeners: {
+      stdout: (data: Buffer) => (versionOutput += data.toString()),
+      stderr: (data: Buffer) => (versionOutput += data.toString())
+    }
+  })
+  const isGnuTar = versionOutput.toUpperCase().includes('GNU TAR')
+
+  // Initialize args
+  const args = [flags]
+
+  let destArg = dest
+  if (IS_WINDOWS && isGnuTar) {
+    args.push('--force-local')
+    destArg = dest.replace(/\\/g, '/')
```

Comment:
Both `destArg` and `file` need the `\` replacement

Reply:
I can add for file too, but when i tested `file` doesnt appear to be required

Reply:
i'll push this change, aesthetically it will look good for consistency